backend/src/openarchiefbeheer/zaken/api/filtersets.py

```diff
@@ -11,6 +11,18 @@
 from ..models import Zaak
 
 
+class ExpandFilter(CharFilter):
+    """
+    Custom filter looking for `field_name` and `lookup_expr` within _expand.
+    """
+
+    def filter(self, qs, value):
+        if value:
+            lookup = f"_expand__{self.field_name}__{self.lookup_expr}"
+            qs = qs.filter(**{lookup: value})
+        return qs
+
+
```

Comment:
Looking at the super method, I think that this is not necessarily needed? 
The filter could be this?

```python
    zaaktype__omschrijving__icontains = CharFilter(
        field_name="_expand__zaaktype__omschrijving", lookup_expr="icontains"
    )
```

Reply:
We could also refactor `_expand__resultaat__resultaattype` to have the same structure :thinking: